docs/lifecycle.rst

```diff
@@ -48,20 +48,20 @@ Message is failed when consume raises any exception or returns negative boolean
             raise Exception("Some issue during create") # exception could be caught at should_retry_cqrs method
 
         @classmethod
-        def cqrs_update(self, sync, mapped_data, previous_data=None):
+        def cqrs_update(cls, sync, mapped_data, previous_data=None):
```

Comment:
Thanks for noticing, actually it's instance method (classmethod decorator is redundant)
```
def cqrs_update(self, sync, mapped_data, previous_data=None):
```